### DISCUSSION TITLE
Começar front end 26 anos

### DISCUSSION THREAD

**Discussion by luizrisk** [frontendbr/forum#1381] [Discussão]
Boa tarde,

Meu nome é Luiz, tenho 26 anos e atualmente trabalho como suporte técnico, faço faculdade de analise e desenvolvimento de sistemas, minha duvida é, estou muito velho para iniciar na area de front end?
Vejo que será necessário estagiar antes, porem com 26 anos casado e com filhos, o salario de um estagio não será suficiente, mas sera difícil alguma empresa contratar sem experiência, pretendia iniciar cursos de html5, css e js para ter o basico, se a vaga de estagio for boa, até posso fazer alguns sacrifícios financeiros, mas um cara de 26 querendo estagiar, vai ter muito garoto de 20 anos que ta na frente do pc desde os 15, devo fazer oq?

**Comment by onlurking** [comment]
> Há alguns anos eu vi em uma comunidade um cara dizendo que já tinha 40 anos e que queria fazer uma graduação na área de TI. Se não me falha a memória, ele disse que o único conhecimento que tinha em informática era windows e pacote office e perguntou o que o pessoal achava dele entrar na área com essa idade.
>
> Lembro que a esmagadora maioria desmotivou o cara, dizendo que tava muito tarde, que ele teria dificuldade em aprender devido a idade e bla bla bla...
>
> Você também acha que não é possível aprender depois dos 40 anos? Então, deixa eu lhe fazer uma pergunta: Você já ouviu falar em "Maurício Samy Silva"? Não? Certamente porque você o conhece pelo seu famoso apelido "Maujor". Sim, ele mesmo "o dinossauro do CSS".
>
> Maurício Samy começou a estudar desenvolvimento web em 1999 aos 56 anos de idade. Em 2003 ele criou o site "http://www.maujor.com/" com o intuito de ensinar CSS, em uma época em que pouquíssimas pessoas conheciam essa tecnologia aqui no Brasil.
>
> Ao longo desses 17 anos, além do seu website (que ainda hoje é referência em CSS), Maurício Samy ministrou cursos, palestras e escreveu nada mais nada menos que 15 livros sobre HTML, CSS, Javascript e Frameworks como Bootstrap, Foundation, Jquery e Compass.
>
> Então, a minha dica é:
>
> Se você acha que não é possível aprender depois dos 40, comece a rever os seus conceitos, porque certamente se você sabe CSS hoje, provavelmente esse conhecimento chegou até você direta ou indiretamente por meio de algum material desenvolvido pelo Maujor.
>
> Não desmotive quem está querendo aprender, temos que além de motivar ajudar e mostrar o melhor caminho para as pessoas aprenderem.
>
> Valeu amiguinhos!!!

- [Fonte](https://pt-br.facebook.com/BugginhoDeveloper/posts/h%C3%A1-alguns-anos-eu-vi-em-uma-comunidade-um-cara-dizendo-que-j%C3%A1-tinha-40-anos-e-qu/571872089649197/)

Então, logo você não está nem um pouco atrasado! Se o caminho pra estágio realmente não for uma opção, a melhor forma de você entrar na área é criar um portfolio, compre cursos de desenvolvimento front-end na Udemy e faça os projetos, coloque eles no ar e participe de eventos (meetups de desenvolvimento na sua cidade), tendo um portfolio e conhecendo outros desenvolvedores você certamente terá uma oportunidade.

**Comment by CassioGangrel** [comment]
Kra eu tô com trinta e estagiando. Acho q idade não é impedimento. Vai fundo se for o que realmente tu deseja. Eu larguei a área de assistência técnica para cursar análise tbm e to estagiando a dois meses.

**Comment by simaodeveloper** [comment]
Mano, comecei com 23 anos, hoje tenho 28, trabalhava e ajudava em casa, não podia fazer estágio pois estagios pagam pouco e eu tinha que ajudar a alimentar a galera de casa, o que eu fiz?

- Iniciei os cursos;
- Me envolvi em comunidades e perguntava muito , claro pesquisava bastante no google e na própria pesquisa dos fórums das comunidades para ver se havia alguém com a mesma dúvida que eu;
- Iniciei projetos pessoais e pequenos, depois fui perguntando quais eram os tipos de projetos que apareciam no mercado e fui atrás de criar alguns parecidos;
- Depois disso adquiri um conhecimento legal e fui atrás de vagas front-end, e nas entrevistas mostrei o que eu construi nessa trajetória;
- Me rodiei de pessoas que queriam a mesma coisa que eu para trocar ideia sobre e consequentemente rolar um networking;

Espero ter te ajudado com a minha vivência, qualquer coisa me chama no telegram **danielsimao**

**Comment by williangd** [comment]
Eu mudei de suporte/redes para desenvolvimento com 31.
Realmente é um pouco difícil encontrar vaga sem experiência, mas isso é difícil para qualquer idade.
Fiz vários cursos online Udemy/Alura/Youtube/etc e alguns projetos pessoais e acabei pegando uma vaga de júnior.
Também acho que a idade não é impedimento

**Comment by fabiochiquezi** [comment]
ótimo post, tbm sempre tive curiosidade de migrar e fazer algumas coisas em áreas além Front-End e tenho esse mesmo receio. 

Lembro de ter visto uma vez histórias de pessoas que se tornaram muito boas no que faziam (desde pinturas, música entre outras coisas), e muitas delas tinham começado depois dos 28 anos, é claro que é algo distante do que o post propõe, estamos falando de mercado e sustento no mundo moderno, mas as vezes serve como inspiração.

**Comment by expalmer** [comment]
Vai firme, comecei com 26 também.
[Veja esse Vídeo do Fabio Akita](https://www.youtube.com/watch?v=VKmPGmFY7H4)

**Comment by joaoricardofl** [comment]
Estou começando na area também, só que eu tenho 34 anos. Acho que qualquer mudança significativa dá um pouco de receio, mas se é o que você realmente quer, vai fundo!

E também, tem várias histórias de pessoas que começaram tarde e realmente se destacaram em algum ramo, por exemplo, o Stan Lee só desenhou o primeiro quadrinho aos 39 anos... 

Acho que em qualquer carreira, sempre vai ter gente mais nova e talentosa, mas acho que tem lugar pra todo mundo, principalmente para aqueles que realmente se esforçam.

**Comment by MarcusVSouza** [comment]
Acabei de fazer 27 e estou na msm jornada, trabalho em uma área totalmente diferente e tô estudando pra mudar, nunca é tarde.

**Comment by Rodrigo54** [comment]
Creio que o problema aqui não seja a capacidade de aprender depois de uma certa idade pois é fato que em indiferente da idade qualquer um pode começar numa carreira nova.

A questão aqui seria mais enquanto a empregabilidade e a preferencia das empresas, pessoas mais jovens tendem a aceitarem salários menores pois estão no inicio da vida adulta e não tem tantas responsabilidades financeiras. 

Mas isso é uma questão que varia muito de empresa pra empresa, a maioria quer economizar com o empregado mas existe também aquelas empresas que valorizam mais a experiencia de vida da pessoa.

Então minha recomendação pra você é que aumente seu networking para se aproximar dessas empresas que vão te ajudar nessa nova jornada.

**Comment by theandersonn** [comment]
@luizrisk Estude e crie um alicerce sólido em base de conhecimento. Cuidado com os hypes, frameworks são excelentes, mas podem ofuscar o que realmente devemos estudar e ter conhecimento sólido.

Tenha como parâmetro você mesmo, o mercado é amplo, tem oportunidades para todos e está carente de pessoas organizadas, focadas.

  Hoje os cursos online são facilitadores, pelas opções e valores. Seja ativo em comunidades, fóruns e eventos presenciais. (_De verdade, faz diferença_).

Ter um GitHub ativo é um diferencial, documente a sua evolução, com o passar de pouco tempo verá a sua evolução, outras pessoas tbm vão ver.

Depois de uma tentativa frustrada, Recomecei 05 anos atrás, estagiário aos 35. Acreditei, estudei muito, tenho estudado muito e a área de desenvolvimento está me surpreendo com ótimos acontecimentos, antes apenas em meus sonhos.

Escrevi, um pouco sobre a trajetória:
https://www.linkedin.com/pulse/de-auxiliar-servi%C3%A7os-gerais-desenvolvedor-web-minha-nascimento/

**Comment by mutheusalmeida** [comment]
Luiz, pesquise sobre a freeCodeCamp. Ela me ajudou bastante e com certeza irá ajudar você também.

**Comment by andreyroberto** [comment]
Achei muito interessante o seu post, hoje me encontro numa situação parecida, estou com 26 anos e cursando o 3° período em Sistemas de Informações, tenho me candidatado a vagas de estágio e com as negativas que tenho recebido, fico me perguntando se estaria "velho" e acredito que isso não influencie, a falta de conhecimentos e também de experiência nesse momento tem um peso grande, mas nada que o esforço não compense.

**Comment by vsalbuq** [comment]
1 - Defina bem o que quer ser (full stack, só front, etc). Eu sei que você não vai ter base suficiente para fazer a melhor escolha possível. Se fizer a escolha certa de primeira, saiba que foi pura sorte. Mas você precisa ter um caminho definido para começar. **É melhor corrigir a rota posteriormente do que não andar pra frente.** Dica: É uma boa olhar bons exemplos de profissionais e tentar saber quanto ele ganha, que cursos fez, quais cargos ocupou antes, quantos anos ficou em cada cargo, etc. Pode perguntar diretamente ou apenas stalkear o linkedIn.

2 - Estude de acordo com o caminho que escolher. Isso pode ser através de um [currículo de estudo próprio](https://github.com/kamranahmedse/developer-roadmap) ou formal. Caso escolha o formal, é bom tentar algo dentro do que as pessoas que você analisou no primeiro passo fizeram. Caminhos comprovadamente bons fornecem alguma garantia relativa. Olhe também as vagas de empresas onde você quer trabalhar. O que eles pedem? Estude isso.

3 - Coloque TUDO o que fizer no Github. É o seu currículo online. À medida que for fazendo coisas melhores, vá excluindo/ocultando seus piores trabalhos. Sempre que puder, veja issues de outros projetos e tente achar soluções, fazendo pull requests. Participe da comunidade. Dependendo de onde você mora, não é incomum se candidatar em alguma empresa e encontrar gente que te viu por aqui.

4 - Desenvolva soft skills. Principalmente se deseja crescer a ponto de liderar equipes. Se quiser uma lista de soft skills, segue [neste link](https://docs.google.com/spreadsheets/d/15irL3QvgTHfk7joqURP8s036njPL4krl6JSw7KksHz4/edit#gid=0). Na primeira aba da planilha, há uma lista. A empresa que a utiliza, usa esse sistema de análise para determinar o valor do salário (que você vê nas linhas de baixo da planilha). Na segunda planilha dessa pasta de trabalho, você encontra uma descrição detalhada de cada soft skill e livros que pode usar para estudar as soft skills listadas.

5 - É uma soft skill, mas merece destaque: Aprenda a se vender. Um bom livro sobre o assunto: Personal Branding.

**Boa sorte!**

Leitura adicional:
- https://woliveiras.com.br/posts/criando-nosso-plano-de-carreira-e-assumindo-o-controle-do-nosso-futuro-profissional/
- https://github.com/woliveiras/front-end-career/blob/master/translations/pt-br/README.md
- https://woliveiras.com.br/posts/criando-nosso-plano-de-carreira-no-trello/

**Comment by mstuttgart** [comment]
@luizrisk pra começar, com 26 anos vocês ainda está jovem haha. Isso seria um empecilho se você desejasse se tornar um atleta profissional ou músico de orquestra (o pessoal começa bem cedo nesses casos)

Eu comecei a programar profissionalmente aos 28 (saindo da faculdade). Como o pessoal comentou, arrumar emprego é difícil em qualquer idade. Somente depois de alguns anos de experiência é que as vagas começam a correr atrás de você.

Eu acredito que nunca é tarde pra aprender algo. Eu tive isso em mente quando comecei e sempre, sempre estudei muito (meu [github](https://github.com/mstuttgart) prova isso :smile: ). Isso me fez passar na frente de muita desenvolvedor que era mais novo. 

> vai ter muito garoto de 20 anos que ta na frente do pc desde os 15, devo fazer oq?

Essa sua frase me lembrou da época que fiz vestibular. No dia da prova tinha centenas de pessoa pra prestar a prova, mas eu sabia que eram poucos ali os que realmente estudavam e que iriam concorrer a uma vaga comigo (digo isso com muita humildade, eu estudava bastante :smile:). 

A dica que te dou é que você deve compensar a idade com esforço e vontade de aprender. Tem muita desenvolvedor jovem por aí, mas são poucos aqueles que realmente aproveitam a pouca idade pra aprender (que são apaixonados pela área). 

Converse com sua esposa (com o apoio da família fica tudo mais fácil), faça as contas e veja qual o valor de salário é necessário pra vocês conseguirem se manter (mesmo fazendo alguns sacrifícios). Eu acredito que, dependendo da empresa, você consegue entrar sem o estágio. A própria empresa te aceita com o objetivo de te treinar (não há nada melhor que uma pessoa motivada a aprender). Então é bom manter o otimismo e mente aberta :smiley: 

Boa sorte!

**Comment by jpferreiradev** [comment]
Putz, a sua historia é um pouco parecida com a minha, estou na área de suporte/Infra e ainda estou na faculdade com 27 anos, doido pra mudar para área de desenvolvimento que eu realmente gosto, principalmente mobile, as opiniões da galera aqui me deram um gás a mais, é bom saber que você não está sozinho correndo atrás de um sonho! Boa sorte na sua jornada o//

**Comment by TiagoFuelber** [comment]
Eu comecei do zero com 27 anos. Hoje estou com trinta e não me arrependo.
Porém no meu primeiro emprego de frontend não foi estágio mas o salário era
baixo, 1.200/mês. Porém eu me dediquei bastante e em um ano e meio depois
já estava ganhando mais que o dobro.
Como não tenho filhos, facilitou.

Em qui, 11 de abr de 2019 00:39, JP Ferreira <notifications@github.com>
escreveu:

**Comment by camila1467** [comment]
nossa.Esse issue me encheu de alegria. To com 22, indo pra 23. No terceiro semestre de bcc e estudando desenvolvimento web, tava me sentindo velha ate..

**Comment by mariojvr** [comment]
Caramba, que issue boa mesmo... Estou com 29 anos, 6 anos de experiência em Administração de Banco de Dados e voltei a a estudar Front. A idade em sí nunca foi um obstáculo para mim mas já desisti várias vezes durante o processo por ficar na dúvida de pegar 6 anos de experiência e "jogado fora" e voltar a ser Junior novamente. Mas quer saber? É isso que eu quero então, bora lá irmão! 
Se quiser trocar alguma ideia sobre estou a sua disposição! Forte abraço a todos!

**Comment by ricardobelfort** [comment]
Fala ai pessoal, cara, eu estou com 37 e migrei da carreira de 18 anos em suporte/infra/redes para a área de desenvolvimento. Já estou a um ano em um instituto que foi o primeiro lugar que me deu a chance de migrar pra área, estou cursando o 3 período de sistemas de informação e não me arrependo de ter migrado de área, pois era o que queria.

Tive medo? Claro que sim, pois sou pai de uma linda princesinha de 9 anos e tenho vários compromissos e responsabilidades. Tive algumas pessoas me incentivando, e foi dai que criei coragem e estou nessa graduação, e um pequeno detalhe (sou formado em economia) desde 2005 mas nunca atuei na área, porque desde aquele tempo eu já estava envolvido com a tecnologia.

Então hoje trabalho como Dev Frontend e amo o que faço! Mas claro, a evolução é contínua, estudando e praticando todos os dias, e esse era o meu medo... de não conseguir aprender por já não ser mais tão novo assim.

Portanto, se você que está com 26 ainda, não tem medo não, vai ser bom pra você, vai em frente e dedique-se bastante. Boa sorte e abraços
